app/components/data/details/data-container-files.component.ts

```diff
@@ -64,4 +63,47 @@ export class DataContainerFilesComponent implements OnDestroy {
             return observable;
         });
     }
+
+    @autobind()
+    public handleFileDelete(files: File[]) {
+        const fileCount = files.length;
+        const taskTitle = `Delete ${fileCount} files from ${this.container.name}`;
+
+        return this.backgroundTaskService.startTask(taskTitle, (task) => {
+            let deleted = 0;
+            const blobCache = this.storageService.getBlobFileCache({
+                container: this.container.id,
+            });
+
+            // NOTE: slight pause in-between deletes to ease load on storage account
+            // may or may not be a great idea.
+            const observable = Observable.interval(100).take(fileCount);
+            observable.subscribe({
+                next: (i) => {
+                    deleted++;
+                    return this.storageService.deleteBlobIfExists(this.container.id, files[i].name).subscribe({
+                        next: (response) => {
+                            task.name.next(`${taskTitle} (${deleted}/${fileCount})`);
+                            task.progress.next(deleted / fileCount * 100);
+
+                            if (response && blobCache) {
+                                blobCache.deleteItem(files[i]);
+                            }
+                        },
+                        error: (error) => {
+                            log.error("Failed to delete blob", error);
+                        },
+                    });
+                },
+                complete: () => {
+                    task.progress.next(100);
+                    // tslint:disable-next-line:max-line-length
+                    const message = `${deleted} files were successfully removed from the file group: ${this.container.name}`;
+                    this.notificationService.success("Removed files from group", message, { persist: true });
```

Comment:
do we really want to persist that one?

Reply:
probably not, though i saw it was in a few other places. can remove.

Reply:
yeah error and warning are persisted by default but info and success disapear.
I force it to displayed for stuff where you have actions that you might want to do